frontend/src/theme/components/Tile.ts

```diff
@@ -0,0 +1,65 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme'
+
+const activeContainerStyle = {
+  bgColor: 'primary.200',
+  borderColor: 'transparent',
+  // NOTE: For the boxShadow styling, due to conflicts with the focus-visible package,
+  // the !important is required to display the boxShadow styling correctly.
+  boxShadow: '0 0 0 3px var(--chakra-colors-primary-500) !important',
+}
+
+const baseContainerStyle = {
+  color: 'inherit',
+  borderRadius: '0.25rem',
+  padding: '1.5rem',
+  _active: activeContainerStyle,
+  _hover: {
+    bgColor: 'primary.100',
+  },
+  _focus: {
+    bgColor: 'white',
+    borderColor: 'transparent',
+    // NOTE: For the boxShadow styling, due to conflicts with the focus-visible package,
+    // the !important is required to display the boxShadow styling correctly.
+    boxShadow: '0 0 0 2px var(--chakra-colors-primary-500) !important',
+  },
+  bgColor: 'white',
+  border: '1px solid',
+  borderColor: 'neutral.300',
+  flexDir: 'column',
+  alignItems: 'flex-start',
```

Comment:
```suggestion
  whiteSpace: 'normal',
  display: 'inline-block',
  wordWrap: 'break-word',
  maxWidth: '20.75rem',
```

Reply:
I think this solves the overflow problem! Just add `textAlign: left` to `title`, `subtitle` and probably add another part for `TileText` to define `textAlign: left` and the color that you've defined inline in the `TileText` component :)

Reply:
ok i think i understand how this happens! from poring over mdn docs, `maxWidth` means exactly that. after the component's `width` exceeds the `maxWidth`, it is deem to have overflowed. 

This then ties into the `whiteSpace` [behaviour](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space), which determines how to layout the elements wrt their whitespace. 

thanks for your help!